### PR TITLE
[Relay][Comment] add a new case for comment deletion with wrong epoch key

### PR DESCRIPTION
**Pull request by jason-ntu**
Please follow the guidelines in [Contribution.md](https://github.com/social-tw/social-tw-website/blob/main/CONTRIBUTING.md) first, then start to create your pull request here.

## Summary

Update `deleteComment` function and the test for it.

## Linked Issue

#258 
#274

## Details
1. Rename a case to "delete the comment failed with wrong proof"
2. Add a case named "delete the comment failed with wrong epoch key"
3. Rename some variables in the test so that it is more accurate.

Evis add
1. Consider `postId` when delete comment

## Impacted Areas

relay

## Tests

relay

## Checklist

-   [x] All new and existing tests pass
-   [x] I have updated the documentation (if applicable)
-   [x] My changes do not introduce new security vulnerabilities
-   [x] Run `yarn lint:fix`

### CODE REVIEW

File: packages/relay/test/comment.test.ts [hunk 1]
```diff
@@ -217,20 +218,45 @@ describe('COMMENT /comment', function () {
             })
     })
 
+    it('delete the comment failed with wrong epoch key', async function () {
+        let epochKeyLiteProof = await userState.genEpochKeyLiteProof({
+            nonce: 1,
+        })
+
+        // invalidate the epochKey
+        epochKeyLiteProof.epochKey = BigInt(0)
```
Comment (Lisooo790926) [inline packages/relay/test/comment.test.ts:227]: This line should be removed, since you will send publicSignals and proof

Comment (Lisooo790926) [reply packages/relay/test/comment.test.ts:227]: And please check the test also, thanks

**Comment by Lisooo790926** [comment]
Hi @jason-ntu 
I also found out the issue when deleting comment, since comment is unique when only given postId and commentId
So could you also help to fix https://github.com/social-tw/social-tw-website/issues/274 in relay level
thanks!!

**Review by Lisooo790926 — APPROVED** [review]

**Review by vivianjeng — APPROVED** [review]
LGTM

**Pull request merged**